package.json

```diff
@@ -67,7 +67,6 @@
     "ember-cli-babel": "^5.1.5",
     "ember-cli-htmlbars": "^1.0.1",
     "ember-basic-dropdown": "^0.9.5-beta.10",
-    "ember-hash-helper-polyfill": "^0.1.0",
```

Comment:
I agree that I not longer need this polyfill, but it concerns me that [this test](https://github.com/cibernox/ember-power-select/blob/master/tests/integration/components/power-select/customization-with-compoments-test.js#L113) is not failing in 1.13 in travis.

Nothing for your to change, just a note to myself.